### DISCUSSION TITLE
Blindness and Hearing Loss should be added as effects

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#6904] [Ideas and suggestions]
I believe that a really good addition to the game would be blindness and hearing loss. These would be effects that go away over time, that would show up in the health monitor, but would not be able to be treated. Blindness should probably make it so that the screen gets darker, and the distance that the player can see decreases. At higher severity, the players vision could even be limited to just a cone of vision, or a black ring around the screen limiting the range that the player can see. Hearing loss would be even more simple, where as it gets stronger, the players range of hearing is decreased (instead of just making sounds quieter, to prevent players from just turning up the volume), and would also distort the sounds as well. After a certain point, the player would only be able to hear very nearby sounds, while it still being very muffled, and after that point, they would loose the ability to hear (instead of making them unable to hear entirely, it could also be done that all sounds are just unrecognizable and quiet, so that there is still some sound). These effects could be given in many different ways, ranging from things like explosions causing blindness and hearing loss, fire causing blindness, nuclear meltdowns causing blindness, concussions causing hearing and blindness, etc. I know hearing loss is kind of a thing when you are in an explosion, but its really just a sound effect, and goes away almost immediately. I think that this would make the game much more interesting, and would also encourage the use of things like helmets and safety equipment, and would also make the punishment for taking damage more long lasting and severe.

**Comment by SpazzMaticus107** [comment]
Hearing loss would be super fun unless it wouldn't go away on it's own or happen too often

**Reply by StruggleDuck** [reply]
Ideally it would go away overtime.
